robocopkaka/bootcamp-29-project
# Fix show spinner when fetching events

#### What does this PR do?
Show a spinner while events are being fetched from the database
#### Description of Task to be completed?
* Show a spinner while events are being fetched from the database
* Add spinners to the AdminProfile and UserProfile components
#### How should this be manually tested?
You should see a spinner on the page while events are being fetched from the database on a page refresh
#### Any background context you want to provide?
The spinner is used to tell the user that an action is being performed
#### What are the relevant pivotal tracker stories?(if applicable)
#155938903
